src/test/java/edu/unc/lib/boxc/migration/cdm/SourceFilesCommandIT.java

```diff
@@ -117,6 +118,37 @@ public void generateBasicMatchDryRunTest() throws Exception {
         assertOutputMatches(".*Total Files in Project: +3.*");
     }
 
+    @Test
+    public void generateDryRunSummaryAndBasicMatchTest() throws Exception {
+        indexExportSamples();
+        addSourceFile("276_182_E.tif");
+        Path mappingPath = project.getSourceFilesMappingPath();
+        Path tempMappingPath = mappingPath.getParent().resolve("~" + mappingPath.getFileName().toString() + "_new");
+
+        String[] args1 = new String[] {
+                "-w", project.getProjectPath().toString(),
+                "source_files", "generate",
+                "--dry-run",
+                "-b", basePath.toString()};
+        executeExpectSuccess(args1);
+
+        assertTrue(Files.exists(tempMappingPath));
```

Comment:
I don't think we want to create an expectation that the temp file is going to exist after a dry run, since we might decide it should be cleaned up later (I almost suggested doing that in an earlier view of this PR). We could instead verify that the regular mappingPath does not exist, since we want to be sure that doesn't show up after a dry run.

This suggestion would apply to all the other dry run tests in this file